homeassistant/components/comelit/__init__.py

```diff
@@ -23,6 +27,22 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return True
 
 
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug("Migrating from version %s", entry.version)
+
+    if entry.version == 1:
+        new_data = entry.data.copy()
+        new_data.update({CONF_PORT: DEFAULT_PORT, CONF_DEVICE: BRIDGE})
+
+        entry.version = 2
```

Comment:
I don't think we need a migration for this case.

Instead, we can set a default when getting it from the config entry data.

Reply:
Removed